static/js/chart/draw_d3_map.ts

```diff
@@ -392,6 +392,7 @@ function addGeoJsonLayer(
  * @param projection projection to use for the map
  * @param zoomDcid the dcid of the region to zoom in on when drawing the chart
  * @param zoomParams the parameters needed to add zoom functionality for the map
+ * @param borderGeoJson the geojson data for drawing borders of containing place
```

Comment:
remove this line in the comments now that it's no longer a param?

Reply:
Thanks for the catch! Removed.